src/config.rs

```diff
@@ -171,6 +171,9 @@ impl<'de> Deserialize<'de> for Keymaps {
                                 de::value::StringDeserializer::new(action),
                             )?;
 
+                            // over-write default key-map to action
+                            navigation.retain(|_, value| value != &ac);
```

Comment:
Is there a less jank way to do this?

Reply:
Looks fine to me, only happens once anyway!